scanner/src/funTest/kotlin/experimental/AbstractProvenanceBasedStorageFunTest.kt

```diff
@@ -188,14 +190,6 @@ abstract class AbstractProvenanceBasedStorageFunTest(vararg listeners: TestListe
     }
 }
 
-private fun ScannerDetails.toCriteria() =
```

Comment:
You have changed the `configMatcher` in your consolidated version: Some tests use a matcher that always returns *true*, while others set an exact matcher. This is probably the reason why multiple tests are failing. You might have to make the matcher configurable in the `ScannerCriteria.forDetails()`.

Reply:
The error was actually in different `maxVersion`s used; I've fixed that.